.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/leastauthority/wormhole-william
 [submodule "magic-wormhole-transit-relay"]
 	path = magic-wormhole-transit-relay
-	url = https://github.com/leastauthority/magic-wormhole-transit-relay
```

Comment:
@vu3rdd What happened here?

Reply:
We were using an old copy off meejah's personal fork. But now, we have merged these changes into the main transit-relay repo, so thought of using it instead of the fork. Once a release is made, we would switch to use that.